DatabaseFileSizeMaintenance.sql

```diff
@@ -13,19 +13,21 @@ License
 -----------
 THE SOFTWARE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 */
-ALTER PROCEDURE [dbo].[DatabaseFileSizeMaintenance]
- @Databases				NVARCHAR(MAX) = NULL
-,@UsedSpacePercentHighThreshold	        INT = 90	-- if used space higher than this, will grow file
-,@UsedSpacePercentLowThreshold	        INT = 10	-- if used space smaller than this, will shrink file
-,@MinFileSizeToShrinkMB			INT = 50000	-- if size smaller than thi  s, will not shrink
-,@MinDatabaseAgeInDays			INT = 30	-- databases must be at least this old to be checked
-,@TargetShrinkSizePercent		INT = 33	-- when shrinking, try to shrink down to this percentage of current file size
-,@MinTargetShrinkSizeMB			INT = 64	-- prevent shrinking to a size smaller than this
-,@ShrinkAllowReorganize			BIT = 1		-- set whether to allow reorganizing pages during shrink
-,@DatabaseOrder				NVARCHAR(max) = NULL
-,@DatabasesInParallel			NVARCHAR(max) = 'N'
-,@LogToTable				NVARCHAR(max) = 'Y'
-,@Execute				NVARCHAR(max) = 'Y'
+ CREATE OR ALTER PROCEDURE [dbo].[DatabaseFileSizeMaintenance]

```

Comment:
@evya696 , please change the command to `ALTER` only.
This is because only newer SQL versions support the `CREATE OR ALTER` syntax, and we want maximum backward compatibility.

Reply:
Thanks